src/App.js

```diff
@@ -1,16 +1,44 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+
 
 const App = () => {
+  const [messageData, setMessageData] = useState(chatMessages);
+  let [totalLikes, setTotalHeart] = useState(0);
```

Comment:
Great initial state values! ✨ 

Your L9, `[messageData, setMessageData]` aligns with React state convention to call the setter value as `set[firstValue]`. To follow the same pattern, L10 should be `[totalLikes, setTotalLikes]`.